x-pack/plugins/security_solution/common/endpoint/constants.ts

```diff
@@ -8,6 +8,7 @@ export const eventsIndexPattern = 'logs-endpoint.events.*';
 export const alertsIndexPattern = 'logs-endpoint.alerts-*';
 export const metadataIndexPattern = 'metrics-endpoint.metadata-*';
 export const metadataCurrentIndexPattern = 'metrics-endpoint.metadata_current-*';
+export const metadataTransformPrefix = 'metrics-endpoint.metadata-current-default';
```

Comment:
I'm stumbling over 3 `-` in here as dataset / namespace should not contain `-`

Reply:
@ruflin 

This is actually the dataset namespace `metrics-endpoint.metadata`. The `current-default` is the name of the transform file as `current-default.json`. I will be moving the transform as per your request to the `metrics-endpoint.metadata_current` and use the filename `default.json`, which means the assets name will be prefixed `metrics-endpoint.metadata_current-default`